package.json

```diff
@@ -48,6 +48,7 @@
     "webpack-dev-server": "^3.7.2"
   },
   "dependencies": {
+    "@openmrs/esm-root-config": "^1.0.0",
```

Comment:
After my first PR review of this code, I released [@openmrs/esm-root-config@1.1.0](https://github.com/openmrs/openmrs-esm-root-config/releases/tag/v1.1.0). Please upgrade to that version. To do so:

```bash
npm install --save @openmrs/esm-root-config
```